wiki/faq.md

```diff
@@ -654,3 +659,25 @@ it can be impossible to send. To fix this, you can try consolidating them using
 try running `consolidateunspent <address>`. After this finishes, try sending
 the transaction again. This may require multiple rounds of the command if you
 have a lot of small transactions.
+
+---
+## Windows Only
+
+---
+
+### Cannot read configuration file. Please check the path and format of the file
+
+First, check you delete the `%appdata%/GridcoinResearch` folder. You should not remove this folder.
+If this folder is not at your `%appdata%`, recreate the folder and add the files `wallet.dat` and `gridcoinresearch.conf` inside the folder.

```

Comment:
I'm not sure if accidentally deleting or moving the folder is going to the most common source of this error

Reply:
I see... I saw this answer searching for this error in Reddit, so I tried to give an answer to this scenario at least, but it makes senses that this message could appear in different scenarios and I could misdirected someone. I will remove it.